usr/share/rear/conf/default.conf

```diff
@@ -334,6 +334,28 @@ CDROM_SIZE=20
 # does not automatically recreate the rescue/recovery system.
```

Comment:
I think it should be mentioned in default.conf
that CHECK_CONFIG_FILES is an automated superset of
what FILES_TO_PATCH_PATTERNS evaluates
(because of the new code in layout/save/default/600_snapshot_files.sh)
so users know that they do not need to manually add files
in FILES_TO_PATCH_PATTERNS to CHECK_CONFIG_FILES.

Reply:
Addressed in 52b28f7b50eb99fbb933e04c5e3be75afeacc894